Library/Formula/ploticus.rb

```diff
@@ -0,0 +1,77 @@
+class Ploticus < Formula
+  desc "Scriptable plotting and graphing utility"
+  homepage "http://ploticus.sourceforge.net/"
+  url "https://downloads.sourceforge.net/project/ploticus/ploticus/2.42/ploticus242_src.tar.gz"
+  version "2.42"
+  sha256 "3f29e4b9f405203a93efec900e5816d9e1b4381821881e241c08cab7dd66e0b0"
+
+  depends_on "libpng"
+
+  # Adds support for PREFIX in Makefile
+  patch :DATA
```

Comment:
I've already sent a note with the patch to the upstream maintainer, but there's nothing to link to: there's no public bug tracker or active forum for Ploticus.

Is "Adds support for PREFIX" not explanatory enough?